BistroQ.Presentation/Views/UserControls/Client/SingleProductControl.xaml.cs

```diff
@@ -53,4 +52,14 @@ private void Button_PointerExited(object sender, PointerRoutedEventArgs e)
     {
         (sender as UIElement)?.ChangeCursor(CursorType.Arrow);
     }
+
+    private void SingleProductGrid_PointerPressed(object sender, PointerRoutedEventArgs e)
+    {
+        ProductClicked?.Invoke(this, Product);
+    }
+
+    private void AddToCartButton_PointerPressed(object sender, PointerRoutedEventArgs e)
```

Comment:
What does this event handler do?

Reply:
I've added a click event to the whole product card to show the detail dialog, so this event handler is to prevent that behavior when clicking "Add" button.